GameEngineTests/GameStateTests.cs

```diff
@@ -76,5 +77,18 @@ public void ShouldNotBeEqualIfTheSunCounterHasMoved()
             Assert.AreNotEqual(initialState.SunCounter, nextState.SunCounter);
             Assert.AreNotEqual(initialState, nextState);
         }
+
+        [TestMethod]
+        public void ShouldUseNextPlayerHandAfterSuccessorIsCalled()
+        {
+            var initialState = TestUtilities.GenerateTestState(playerCount: 2);
```

Comment:
I have a preference for using 3 players here so we're testing a more interesting case, but it's not a big deal.

Reply:
👍